### DISCUSSION TITLE
view使用iconfont图标，通过点击事件改变图标内容，但是不能正确显示为图标

### DISCUSSION THREAD

**Discussion by edward-chung** [BYJRK/DotNet-Discussions#29] [Q&A] [answered]
`<StackPanel>`
`    <TextBlock x:Name="tb" Text="&#xe672;" FontSize="18" FontFamily="/fonts/#iconfont" />`
`    <Button Content="改变图标" Click="Button_Click" />`
`</StackPanel>`

`private void Button_Click(object sender, RoutedEventArgs e)`
`{`
`    tb.Text = "&#xe671;";`
`}`

![Snipaste_2023-02-16_13-23-41](https://user-images.githubusercontent.com/9280859/219275986-6366602a-ac2b-47d4-afb3-50aeaba67c00.png)
![Snipaste_2023-02-16_13-23-53](https://user-images.githubusercontent.com/9280859/219275999-2d5c951f-58fe-4f46-adbd-54fd86da158b.png)

大佬，这个问题怎么解决，求教

**Comment by KiyanYang** [comment, marked as answer]
改为
```
tb.Text = "\xe671";
```

**Reply by BYJRK** [reply]
对的，`&#;` 是 XML 里面使用的，后台要用 C# 的十六进制表示方式

**Comment by edward-chung** [comment]
原来如此，感谢大佬
